Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Pumps/ChilledWater/Controller.mo

```diff
@@ -49,64 +49,73 @@ block Controller
   final parameter Integer pumInd[nPum]={i for i in 1:nPum}
     "Pump index, {1,2,...,n}";
 
-  Buildings.Controls.OBC.CDL.Interfaces.IntegerInput uPumLeaLag[nPum]
+  Buildings.Controls.OBC.CDL.Interfaces.IntegerInput uPumLeaLag[nPum] if
```

Comment:
It is a bit unclear to me what this input represents. Can you please clarify, as we now have lead and lag as categories, and the values are integers. Is it that the first value in the array is the index of the lead device?

Reply:
The array should be `[lead pump index, 1st lag pump index, 2nd lag pump index, ...]`